### PR TITLE
[SCB-245]: add acceptance test for dubbo demo

### PR DESCRIPTION
**Pull request by wuzunqian**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

I add acceptance test for dubbo demo, this change base on branch [SCB-547].

### CODE REVIEW

File: saga-demo/saga-dubbo-demo/service-pub/src/main/java/org/apache/servicecomb/saga/demo/dubbo/pub/ServiceVO.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package org.apache.servicecomb.saga.demo.dubbo.pub;
```
Comment (WillemJiang) [inline saga-demo/saga-dubbo-demo/service-pub/src/main/java/org/apache/servicecomb/saga/demo/dubbo/pub/ServiceVO.java]: License header please.

**Comment by yangbor** [comment]
The PR title format is incorrect. Please change it to [SCB-245] Add acceptance test ...

**Comment by WillemJiang** [comment]
We need to start the service and the db before running the cucumber test.

**Comment by wuzunqian** [comment]
I think It's same with acceptance-pack?

**Comment by WillemJiang** [comment]
@wuzunqian  yeah, you r right.  In this way we can run the test by using mvn clean install.

File: omega/omega-transport/omega-transport-dubbo/src/test/java/SagaDubboProviderFilterTest.java [hunk 2]
```diff
@@ -45,7 +45,6 @@
   @Before
   public void setUp() {
     omegaContext.clear();
-    ApplicationContext applicationContext = mock(ApplicationContext.class);
```
Comment (WillemJiang) [inline omega/omega-transport/omega-transport-dubbo/src/test/java/SagaDubboProviderFilterTest.java]: JUnit always creates a new mock applicationcontext instance for the member field.  But this applicationContext is a different scope.  There is some miss part need to be add.

Comment (wuzunqian) [reply omega/omega-transport/omega-transport-dubbo/src/test/java/SagaDubboProviderFilterTest.java]: yes, I have fix it at this commit : 2fda9f6, but The Travis CI build failed

**Comment by WillemJiang** [comment]
@wuzunqian  
I got the dubbo registry error when running the application with spring-boot-2. Not sure if dubbo 2.6.1 supports Spring boot 2. Now I had to disable the test when running the accept test with spring boot 2. 

Caused by: com.alibaba.dubbo.rpc.RpcException: Failed to register dubbo://172.20.10.6:20881/org.apache.servicecomb.saga.demo.dubbo.api.IServiceA?anyhost=true&application=dubbo-servicea&dubbo=2.6.1&generic=false&interface=org.apache.servicecomb.saga.demo.dubbo.api.IServiceA&methods=cancelRun,run&pid=63137&revision=0.2.0-SNAPSHOT&side=provider&timestamp=1527315694864 to zookeeper zookeeper://localhost:2181/com.alibaba.dubbo.registry.RegistryService?application=dubbo-servicea&client=zkclient&dubbo=2.6.1&interface=com.alibaba.dubbo.registry.RegistryService&pid=63137&qos.enable=false&qos.port=33333&timestamp=1527315691973, cause: Zookeeper is not connected yet!

**Comment by WillemJiang** [comment]
It turns out zookeep connection error is caused by the class not found exception of java.lang.ClassNotFoundException: org.apache.log4j.Logger 
I did a quick fix on the branch of SCB-555, we can merge it to master branch once we got your iCLA.
@wuzunqian  Please check out the changes and feel free to add your comments.

**Comment by WillemJiang** [comment]
Got the iCLA from wuzunqian.  I just created new PR #194  which is based on this PR.

**Comment by WillemJiang** [comment]
The PR is a part of #194 which is merged into the master branch, now we can close this PR.

**Pull request closed without merging**